scripts/sct_propseg.py

```diff
@@ -297,11 +297,11 @@ def get_parser():
     parser = get_parser()
     arguments = parser.parse(sys.argv[1:])
 
-    input_filename = arguments["-i"]
+    fname_data = arguments["-i"]
```

Comment:
For safer use of the fname_data, normalizing it will avoid needing to check the format.
`fname_data = os.path.abspath(arguments['-i'])`